CWE/events/trieste.txt

```diff
@@ -1,88 +1,3 @@
-#Establish the Free City of Trieste
```

Comment:
These events are necessary.

Reply:
If they are necessary I will have to tie them to the Paris Peace Treaty and have them reviewed as OTL it had been made clear since before the end of the war that the territory was to be established and was a clause of the treaty

Reply:
Okay but please tie them by making the Paris peace treaty event set a global flag for this event to fire instead of deleting these events.